integrations/docker/images/stage-2/chip-build-efr32/Dockerfile

```diff
@@ -30,8 +30,8 @@ RUN git clone --depth=1 --single-branch --branch=2.8.2 https://github.com/Silico
     rm -rf .git \
     && : # last line
 
-# Clone WiSeConnect SDK 3.1.0 (58a2404)
-RUN git clone --depth=1 --single-branch --branch=v3.1.0 https://github.com/SiliconLabs/wiseconnect.git /tmp/wifi_sdk && \
+# Clone WiSeConnect SDK 3.1.1 (b2c1cd0)
```

Comment:
If you are updating the docker file, it is required you update the docker image version. 

That also confuse me that the CI is passing. As this new docker image doesn't exist yet for the CI. Are you not using wise connect for docker for the CI? If so then why add it to the docker file?


Usually the docker update is done in a separate PR that needs to be merged before the one that updates the code. Else the docker image isn't yet available for the CI